server/main_chatbot.py

```
@@ -1452,19 +1462,21 @@ async def confirmation_sender(data):
                         "type": "error",
+                        "session_id": session_id,
                         "data": {
                             "text": f"Your message is too long ({input_token_count} tokens). Please limit your message to 20,000 tokens (approximately 80,000 characters).",
                             "severity": "error",
                             "session_id": session_id,
                         }
                     }))
                     continue  # Skip processing this message
-                    
+
             except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)